managedplugin/download.go

```diff
@@ -154,6 +154,22 @@ func doDownloadPluginFromHub(ctx context.Context, c *cloudquery_api.ClientWithRe
 	case http.StatusUnauthorized:
 		return fmt.Errorf("unauthorized. Try logging in via `cloudquery login`")
 	case http.StatusNotFound:
+		// See if the plugin exists, but not the version.
+		pvw, err := NewPluginVersionWarner(logger, ops.AuthToken)
+		if err != nil {
+			return fmt.Errorf("failed to create plugin version warner: %w", err)
+		}
+
+		ver, err := pvw.getLatestVersion(ctx, ops.PluginTeam, ops.PluginName, ops.PluginKind)
```

Comment:
Similar here, since probably the best thing we can tell users is to run `cloudquery login` in case of an error I'll keep the [original one](https://github.com/cloudquery/plugin-pb-go/pull/423/files#diff-65f60ea2b094fb37ad7333772b6c3708cd026ed6729984bc77f8878de444bc65R173)

Reply:
Done